manifest.toml

```diff
@@ -38,6 +38,9 @@ ram.runtime = "50M"
     type = "group"
     default = "visitors"
 
+    [install.admin]
+    type = "user"
```

Comment:
(i'm a bit confused about where that `admin` variable is used, but if it is relevant it should probably be auto-added in the upgrade script)

Reply:
I was using `admin` in the config file but removed the section. I guess we can remove it.